.well-known/nostr.json

```diff
@@ -3,6 +3,7 @@
         "utxoclub": "91dbab9f62660e95258480d2f2cff6dcfdb513f28a85fa4fb55ee993a5b46809",
         "crispy": "a9bf5670ce5d9affb89117290a7526e32f3cf6d964a882828920ed2acd363fa6",
         "tigs": "079d727bee3d805e24758861ea4d028b2e6a37805f8db45bf8045bf900099650",
-        "sidshattuck": "72f22deb08d9767c250aaab1979ebb46eaea08c86d9ac4c59e476f68545fa81a"
+        "sidshattuck": "72f22deb08d9767c250aaab1979ebb46eaea08c86d9ac4c59e476f68545fa81a",
+        "aurelian_btc": "npub1szwhg8dqep7wl87whnr5xukxskt8vgghrn9u8k9shjxc7jjh6xfsm8vvt2"
```

Comment:
Actually. Is it supposed to have `npub` at the start? The others don't.

Reply:
Doh! Should be in hex